python/pyspark/sql/tests/test_session.py

```diff
@@ -225,6 +225,24 @@ def test_get_active_session_after_create_dataframe(self):
                 session2.stop()
 
 
+class SparkSessionTests4(ReusedSQLTestCase):
```

Comment:
SparkSessionTests5?

Reply:
@afavaro can you address this comment? I think it's good to go.

Reply:
Done